content/blog/zardshard/2023-05-05_[gsoc_2023]_plans_for_improving_icon-o-matic.md

```diff
@@ -0,0 +1,47 @@
++++
+type = "blog"
+title = "[GSoC 2023] Plans for improving Icon-O-Matic"
+author = "Zardshard"
+date = "2023-05-05 14:07:09-04:00"
+tags = ["haiku", "software", "icon-o-matic", "gsoc", "gsoc2023"]
++++
+
+Icon-O-Matic is the editor for HVIF files, the format that Haiku's icons are in. It's a relatively simple but impressive aplication. It does have room for improvement, however.
+
+There are three things I am planning to do during GSoC: fix bugs, refactor the code, and improve the UI. There is a list of known bugs that Icon-O-Matic has over on [Trac](https://dev.haiku-os.org/query?component=%5EApplications%2FIcon-O-Matic&status=assigned&status=in-progress&status=new&status=reopened&order=priority&col=id&col=summary&col=status&col=type&col=priority&col=milestone&col=component). You can help, too, by reporting bugs in Icon-O-Matic over there. Less obviously, you can also vote for tickets that you would like to be resolved. To do this, click on the ticket that interests you, then click one of the buttons highlighted by the arrow: ![The buttons are in the upper-right hand corner among the navigation links](/files/blog/zardshard/trac_upvoting.png)
```

Comment:
```suggestion
There are three things I am planning to do during GSoC: fix bugs, refactor the code, and improve the UI. There is a list of known IOM bugs over on [Trac](https://dev.haiku-os.org/query?component=%5EApplications%2FIcon-O-Matic&status=assigned&status=in-progress&status=new&status=reopened&order=priority&col=id&col=summary&col=status&col=type&col=priority&col=milestone&col=component). You can help, too, by reporting bugs in Icon-O-Matic over there. Less obviously, you can also vote for tickets that you would like to be resolved. To do this, click on the ticket that interests you, then click one of the buttons highlighted by the arrow: ![The buttons are in the upper-right hand corner among the navigation links](/files/blog/zardshard/trac_upvoting.png)
```

Reply:
I don't think abbreviating IOM is necessary here. If it were said too often, then I would want to abbreviate it.